src/ILVerify/src/Program.cs

```diff
@@ -82,6 +78,7 @@ private ArgumentSyntax ParseCommandLine(string[] args)
                 syntax.HandleErrors = true;
 
                 syntax.DefineOption("h|help", ref _help, "Help message for ILC");
+                syntax.DefineOption("s|system-module", ref _systemModule, "Simple name of the system module for compilation.");
                 syntax.DefineOptionList("r|reference", ref referenceFiles, "Reference file(s) for compilation");
```

Comment:
Sorry, I didn't see that the hand crafted help text was actually printing different help descriptions for the options.
I will change the option help strings to the previous (hand crafted) help texts.